SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2699,6 +2699,26 @@ bool USpatialNetDriver::IsLogged(Worker_EntityId ActorEntityId, EActorMigrationR
 	return bIsLogged;
 }
 
+int64 USpatialNetDriver::GetClientID()
+{
+	if (IsServer())
+	{
+		return (int64)SpatialConstants::INVALID_ENTITY_ID;
+	}
+
+	if (USpatialNetConnection* NetConnection = GetSpatialOSNetConnection())
+	{
```

Comment:
Actually, PlayerControllerEntity is already available on the NetConnection. I believe this is set correctly on clients also.